doc/reference/reference_lua/errcodes.rst

```diff
@@ -1,112 +1,135 @@
 .. _error_codes:
 
--------------------------------------------------------------------------------
 Database error codes
--------------------------------------------------------------------------------
+====================
 
 In the current version of the binary protocol, error messages, which are usually
 more descriptive than error codes, are not present in server responses. The actual
```

Comment:
Should we update the description? It seems error messages are passed just fine over the binary protocol.
Example:
```
tarantool> nb=require'net.box'
---
...

tarantool> c=nb.connect(3301)
---
...

tarantool> ok,err = pcall(c.eval, c, "box.schema.space.create('test', {is_local=true, is_sync=true})")
---
...

tarantool> err:unpack()
---
- code: 9
  base_type: ClientError
  type: ClientError
  message: 'Failed to create space ''test'': local space can''t be synchronous'
  trace:
  - file: ./src/box/alter.cc
    line: 523
...


```

Reply:
Thanks! I had doubts regarding this info but didn't check by myself. Will fix.

Reply:
Removed this information from the intro:
https://docs.d.tarantool.io/en/doc/box-error/reference/reference_lua/errcodes/
Looks like this info might be added to the net.box help topic.